frontend/src/ts/url.ts

```diff
@@ -139,6 +139,9 @@ export const buildReshiftLink = (
   type: string
 ) => {
   if (region.startsWith('cn')) {
+    if (type === 'serverless') {
```

Comment:
Let's simple this method like https://github.com/awslabs/clickstream-analytics-on-aws/pull/1292/files#diff-45f5cd9145b11ab8e5eff90e9b8fe9d6cf714d10e35730ec8217de521c626f64R198

Reply:
done